src/DataflowType/AbstractDataflowType.php

```diff
@@ -24,12 +27,22 @@ public function getAliases(): iterable
     public function process(array $options): Result
     {
         $optionsResolver = new OptionsResolver();
+        $optionsResolver->setDefault('type', self::TYPE_SYNC);
```

Comment:
Risky, as existing dataflow types might already be using `type` as an option. That's why I suggested using a protected method instead.

Basically :
`$builder = $this->createDataflowBuilder();`

```
protected function createDataflowBuilder(): DataflowBuilder
{
    return new DataflowBuilder();
}
```

And then the concrete dataflow can just override that method to use the builder it wants.